Source/Core/Core/HW/GCMemcardDirectory.cpp

```diff
@@ -266,9 +313,8 @@ s32 GCMemcardDirectory::Write(u32 destaddress, s32 length, u8 *srcaddress)
 
 		// verify that we haven't calculated a length beyond BLOCK_SIZE
 		_dbg_assert_msg_(EXPANSIONINTERFACE, (destaddress + length) % BLOCK_SIZE == 0,
-						 "Memcard directory Write Logic Error");
+			"Memcard directory Write Logic Error");
```

Comment:
Not that I really care, but judging from GitHub's display, neither of these indentations is right.  Keep the same tab level as the previous line and use spaces for the rest.